python/benchmarks/oxford_pet/train.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+"""Train and evaluate models on Oxford pet dataset.
+
+"""
+
+import os
+from typing import Callable, Optional
+
+import click
+import pytorch_lightning as pl
+import torch
+import torchvision
+import torchvision.transforms as T
+from common import Classification, RawOxfordPetDataset, collate_fn, raw_collate_fn
+from pytorch_lightning.loggers import TensorBoardLogger
+from torchdata.datapipes.iter import IterableWrapper
+from torchvision.transforms.functional import InterpolationMode
+
+import lance
+import lance.pytorch.data
+
+NUM_CLASSES = 38
+
+
+class TrainTransform(torch.nn.Module):
+    """
+    Image transform for training.
+
+    Adding random argumentations.
+
+    https://github.com/pytorch/vision/blob/a89b1957a62e2f68f001d5d60268743edbe164d8/references/classification/presets.py#L6
+    """
+
+    def __init__(
+        self,
+        crop_size: float,
+        mean: tuple[float] = (0.485, 0.456, 0.406),
```

Comment:
where do these defaults come from?

Reply:
These numbers are used across torchvision, and used in the python code referred above.